Library/Formula/collectd.rb

```diff
@@ -19,6 +16,17 @@ class Collectd < Formula
     depends_on "autoconf" => :build
   end
 
+  stable do
+      url "https://collectd.org/files/collectd-5.5.0.tar.bz2"
+      mirror "http://pkgs.fedoraproject.org/repo/pkgs/collectd/collectd-5.5.0.tar.bz2/c39305ef5514b44238b0d31f77e29e6a/collectd-5.5.0.tar.bz2"
+      sha256 "847684cf5c10de1dc34145078af3fcf6e0d168ba98c14f1343b1062a4b569e88"
+
+      patch do
```

Comment:
I use vim. These are spaces. Just 4 instead of 2 -- I use `ts=4 sw=4` as my default. Does homebrew adhere to a 2 space style guide?